content/reference/endpoints/bookings_taxes.md

```diff
@@ -0,0 +1,43 @@
+# Bookings Taxes
+
+1. TOC
+{:toc}
+
+### Parameters
+<ul class="nav nav-pills" role="tablist">
+  <li class="disabled"><a>OAuth Scopes:</a></li>
+  <li class="active"><a href="#public" role="tab" data-toggle="pill">public</a></li>
+</ul>
+<div class="tab-content" markdown="1">
+  <div class="tab-pane active" id="public" markdown="1">
+Name             | Type    | Read/Write | Description
+-----------------|---------|------------|------------
+id               | Integer | Read       | Bookings Tax's id.
+-----------------|---------|------------|------------
+amount           | [Decimal](/reference/enums#formats)  | Read       | Bookings Tax's amount.
+name             | [Object](/reference/enums#formats)   | Read       | Bookings Tax's name, list of locales described in [enums section](/reference/enums#locales).
+percentage       | [Decimal](/reference/enums#formats)  | Read       | Bookings Tax's percentage.
+tax_included_in_price | Boolean | Read       | Tax amount already included in bookings price if set to true.
```

Comment:
:D

So we've got Booking with Rental's price 100 (initial price) with included VAT 23% and some kind of Fee with price 100, but the Tax is not included, so the final price will be 100 + 100 + 100 * 23%, 2 BookingsTaxes will be created: One for the Rental (with amount 100 / 1.23), as it's included in price and for BookingsFee with amount 100 * 0.23 which is not included in price, but the inclusion is about Taxable's price (Rental or BookingsFee), not the Booking's.